Source/WebCore/platform/graphics/Path.h

```diff
@@ -92,6 +92,7 @@ class Path {
     std::optional<PathDataBezierCurve> singleBezierCurve() const;
 
     WEBCORE_EXPORT bool isEmpty() const;
+    bool isSingleDataLine() const;
```

Comment:
Not clear what a "data line" is from outside of Path. Could this just be `isSingleLine`?

Reply:
Sounds good. Changed.